README.md

```diff
@@ -50,10 +50,27 @@ perform authenticated HTTP requests.
 
     signer = atlassian_jwt_auth.create_signer('issuer', 'issuer/key', private_key_pem)
     response = requests.get(
-        'https://your-url'
+        'https://your-url',
         auth=JWTAuth(signer, 'audience')
     )
 ```
+One can also use `atlassian_jwt_auth.contrib.aiohttp.JWTAuth`
+to autenticate `aiohttp` requests:
```

Comment:
Pro tip: maintainers normally have write access to the PR branch in contributor repository.